src/core/build.jl

```diff
@@ -12,9 +12,22 @@ function _instance_datatype(md::ModelDef, def::DatumDef, start::Int)
     elseif dims[1] != :time
         T = Array{dtype, num_dims}
     
-    else
-        step = step_size(md)
-        T = TimestepArray{dtype, num_dims, start, step}
+    else   
+        # OLD WAY
+        # step = step_size(md)
+        # T = TimestepArray{dtype, num_dims, start, step}
```

Comment:
Lets remove old stuff like that. We can always look at the git history if we want to see how it was done previously, but this kind of stuff tends to accumulate, I think it is better to just delete it.